desktop/flipper-ui-core/src/chrome/SettingsSheet.tsx

```diff
@@ -499,6 +513,46 @@ export default connect<StateFromProps, DispatchFromProps, OwnProps, Store>(
   {updateSettings, updateLauncherSettings},
 )(withTrackingScope(SettingsSheet));
 
+function AndroidUserIdField(props: {
+  defaultValue: string;
+  onChange: (path: string) => void;
+}) {
+  const activeDevice: BaseDevice = useSelector(getActiveDevice);
+  const [users, setUsers] = useState([] as {id: string; name: string}[]);
+
+  activeDevice
+    .executeShell('pm list users')
+    .then((result) => {
+      const users = result
+        .match(/(?<=UserInfo{)(.*?)(?=})/g)
+        ?.map((userInfo) => {
+          const infos = userInfo.split(':');
+          return {id: infos[0], name: `${infos[0]} (${infos[1]})`};
+        });
+      setUsers(users || []);
+    })
+    .catch((error) => console.error(error));
```

Comment:
Could you add some more details to the error message? Also let's start with `.warn`, as this is typically a local setup issue

Reply:
This command should not fail. If this command fails there is chances that every shell command will. If I have a look on Flipper source code where executeShell is called, errors are not even managed. For example, on plugins/public/cpi/index.tsx

const chipname = await executeShell('getprop ro.chipname');

There is no try/catch.
The only reason why I've got a catch it's because I don't know how to await in a React context so linters are compelling me to consume the promise in the good old way (.then/.catch) with a compelled catch block (as per linters rules).

With these informations in mind, what do you think about chances I should write on this? Just a warn instead of an error?